pkg/metrics/certificates_test.go

```diff
@@ -422,77 +444,72 @@ func TestCertificateCache(t *testing.T) {
 	}
 
 	// Remove second certificate and check not exists
-	m.RemoveCertificate(types.NamespacedName{
-		Namespace: "default-unit-test-ns",
-		Name:      "crt2",
-	})
-	if err := testutil.CollectAndCompare(m.certificateReadyStatus,
+	err = certsInformer.Informer().GetIndexer().Delete(crt2)
+	assert.NoError(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)

```

Nit: To make the code a bit more readable.